Documentation/ldap-connector.md

```diff
@@ -88,6 +88,81 @@ connectors:
 The LDAP connector first initializes a connection to the LDAP directory using the `bindDN` and `bindPW`. It then tries to search for the given `username` and bind as that user to verify their password.
 Searches that return multiple entries are considered ambiguous and will return an error.
 
+## Example: Mapping a schema to a search config
+
+While adapting an existing LDAP schema to a connector config, use a tool such as `ldapsearch` for exploration. These tools display entries and attributes to help compose user and group searches.
+
+For smaller directories it may be practical to dump the entire contents and search by hand.
+
+```
+ldapsearch -x -h ldap.example.org -b 'dc=example,dc=org' | less
+```
+
+First, find a user entry. User entries declare users who can login to LDAP connector using username and password.
+
+```
+dn: uid=jdoe,cn=users,cn=compat,dc=example,dc=org
+cn: Jane Doe
+objectClass: posixAccount
+objectClass: ipaOverrideTarget
+objectClass: top
+gidNumber: 200015
+gecos: Jane Doe
+uidNumber: 200015
+loginShell: /bin/bash
+homeDirectory: /home/jdoe
+uid: janedoe
+```
+
+Compose a user search which returns this user.
+
+```yaml
+userSearch:
+  # The directory directly above the user entry.
+  baseDN: cn=users,cn=compat,dc=example,dc=org
+  filter: "(objectClass=posixAccount)"
+
+  # Expect user to enter "janedoe" when logging in.
+  username: uid
+
+  # Use the full DN as an ID.
+  idAttr: DN
+
+  # When email is unavailable, it's recommended to use an equally unique value.
+  emailAttr: uid
+  nameAttr: gecos
+```
+
+Second, find a group entry. The following is an example of a group return by a FreeIPA server:
+
+```
+dn: cn=developers,cn=groups,cn=compat,dc=example,dc=org
+memberUid: janedoe
+memberUid: johndoe
+gidNumber: 200115
+objectClass: posixGroup
+objectClass: ipaOverrideTarget
+objectClass: top
+cn: developers
+```
+
+Group searches are a bit harder, since they need to match an attribute on the user entry to a group entry attribute. For this example, the search should match the `uid` user attribute to the `memberUid` group attribute.
```

Comment:
Replace with: "Group searches must match a user attribute to a group attribute. In this example, the search returns users whose `uid` is found in the group's list of `memberUid` attributes."